src/postgraphile/cli.ts

```diff
@@ -278,7 +278,7 @@ pluginHook('cli:flags:add:webserver', addFlag);
 program
   .option(
     '-e, --jwt-secret <string>',
-    'the secret to be used when creating and verifying JWTs. if none is provided auth will be disabled',
+    'the secret or private key to be used when creating and/or verifying JWTs. If none is provided, JWT auth will be disabled. If using a private key, the value should be in the PEM format and the public key should be provided via `--jwt-public-key`',
```

Comment:
The `--jwt-public-key` flag doesn't seem to have been added.

Reply:
I was sure we've decided in our previous conversation to avoid adding new CLI parameters and use config file instead.

Reply:
In which case it needs removing from the option description :)

Reply:
Sorry, seems like I'm confused with the Github's PR interface... Will update sources

Reply:
Updated. But CI build is failing, I don't understand why...

Reply:
It could certainly be more obvious! (PR to improve that would be welcome.)

The issue is this:


```
$ yarn prettier --list-different
$ prettier 'src/**/*.[tj]s' 'postgraphiql/src/**/*.[tj]s' --list-different
src/postgraphile/withPostGraphileContext.ts
```

This means that `src/postgraphile/withPostGraphileContext.ts` has not been formatted with prettier. You should be able to fix it by running `yarn prettier:fix`. My editor runs prettier for me automatically; I don't know how to format code by hand any more :wink:

Reply:
I just applied your suggested change in the GitHub UI and it failed the prettier check. That's why I was confused as I remember that I didn't change this file on my own.

Reply:
So, now prettier is happy, but something went wrong in the `package.json`...

Reply:
This is not the fault of your PR, it's now passing CI. However looks like a vulnerability has been discovered in lodash in the mean time.

The easiest way to fix this is to delete these four lines from the `yarn.lock`:

https://github.com/graphile/postgraphile/blob/d094541ab63d9f5cab9ac6ef36d37b8ca09f054d/yarn.lock#L5006-L5009

And then run `yarn` again. This should install a newer version of lodash that fixes the vulnerability and appeases Snyk. We don't use `defaultsDeep` so we're technically not vulnerable. Don't worry if you don't want to take this on, I can address it on master next week and then merge it back into this branch.